src/open-people.ejs

```diff
@@ -21,12 +21,25 @@
                         </div>
                         <div class="l-desktop-grid-cell l-grid-cell">
                             <ul class="base-list">
+                                <li class="trailing-margin">
+                                    <h3 class="alt-h3"><a href="https://docs.google.com/document/d/e/2PACX-1vSyvQgYaTqAgaiaORShaqhWod2MsyS3iugypeBjyxP4Y9i676DcYvzgBH4YHYl-l8p3A8IU2UOA14Jr/pub">
+                                            How we promote engineers
+                                        </a></h3>
+                                    <p>We use this internally to ensure every engineer in the department understands
+                                        our career progression process, and for that process to be clear and consistent to everyone.<br>
+                                        There may seem to be a lot of detail there, but this is because we take a lot of
+                                        effort to make it fair and consistent.
+                                    </p>
+                                    <p>In the spirit of keeping our process up to date with the needs of the team, we are
+                                        making our career progression work better for everyone.  Expect some changes here in the coming
+                                        months.</p>
+                                </li>
                                 <li class="trailing-margin">
                                     <h3 class="alt-h3"><a href="https://docs.google.com/spreadsheets/d/e/2PACX-1vRqPMDtzJTvV1V42JyM4R_koBqnsoM9CY-yA2lGJF6c5tIpYVPVZwOnlGgQz2m7inar3_eSjiKlcww3/pubhtml">
-                                            Career progression framework
+                                            Career progression grid
```

Comment:
I thought for external reading, grid was more clear in terms of distinction from the process, but opinions welcome, happy to change back.